src/main/scala/com/cloudera/sparkts/TimeSeries.scala

```diff
@@ -231,6 +232,37 @@ class TimeSeries[K](val index: DateTimeIndex, val data: DenseMatrix,
   }
 
   /**
+    * This function applies the filterExpression on each row (instant)
```

Comment:
It might be clearer to say that the function is applied to each element of each row?